readme/Releases.md

```diff
@@ -117,4 +117,6 @@ followed when publishing a release:
   * [ ] Update the release description.
   * [ ] Test the release draft artifacts for obvious issues.
   * [ ] Publish the release.
+* [ ] Check that the FlatHub release is built successfully.
```

Comment:
If mentioning the FlatHub release, you might want to specify the repo 
https://github.com/flathub/com.github.xournalpp.xournalpp

Also you could mention the SnapCraft release initiated via a PR to https://github.com/kenvandine/xournalpp

Reply:
The SnapCraft release is unofficial, I think this is fine as-is.